jest.config.js

```diff
@@ -2,4 +2,5 @@ module.exports = {
   preset: '@stoplight/scripts',
   setupTestFrameworkScriptFile: './setupTests.js',
   snapshotSerializers: ['enzyme-to-json/serializer'],
+  collectCoverageFrom: ["<rootDir>/src/**/*.{ts,tsx}"]
```

Comment:
@marbemac I think it'd be good to have `.tsx` in shared config.

Reply:
Done! For your review and merge over here: https://github.com/stoplightio/scripts/pull/13